### DISCUSSION TITLE
Cannot see the current size of the playing field if scaling display is used

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#17606] [Ideas] [answered]
The current state of the screen scaling feature has several drawbacks. I want to pay attention to them:
1. It is not possible to set a specific screen resolution when scaling the screen if you are in fullscreen mode. I want to set 1600x900 and I only have to do it roughly. This problem has a simple solution - show the window size, as shown when resizing the application itself (You can also add a text field where you can manually specify the screen resolution). Example:
![osu_2022-04-01_21-34-14](https://user-images.githubusercontent.com/59089206/161322659-877755b5-18f6-4a5e-9f04-dcda6246c274.jpg)
2. Moved into https://github.com/ppy/osu/discussions/16828#discussioncomment-2490340
3. Moved into new topic - https://github.com/ppy/osu/discussions/17611

**Comment by peppy** [comment, marked as answer]
Please split up into individual discussions (you can edit this discussion down to just one discussion point). Also please search first if you didn't, I'm 99% sure some of these are duplicated.

**Reply by Moltanicaa** [reply]
Did this

**Comment by peppy** [comment]
To confirm, did you mean "set" in the issue title, not "see"?

If that is what you meant to write, then I agree that playfield scaling should likely be moved out to its own setting so it can be applied in addition to the screen level scaling.

**Reply by Moltanicaa** [reply]
I wrote correctly, I don't see the exact size of the area I set.  I want to set the size of the playing field as 1600x900, but I have to do it approximately.

**Reply by peppy** [reply]
I'm confused then. Your screenshot looks to be photoshopped so I'm not sure what it's intending to show, but to confirm:

- You're using "excluding overlays" scaling
- You want to see how large the playfield is

You can already do this by running autoplay, correctly? Are you somehow wanting to set exact pixel dimensions of the *playfield* while applying *screen scaling*? This is not really something we can or will support, because it will change relative to whatever you do to the window.

**Reply by Moltanicaa** [reply]
Exactly this. I showed on the screen that the problem has a simple solution (as it seems to me). It is possible that the misunderstanding is due to the fact that this setting item goes along with the third item of my topic.

The main problem is that due to the lack of the ability to set the exact dimensions of the field, as well as the lack of a semblance of native rendering, a situation arises that I described in the link to the third topic (the aim drops very much)

**Reply by Moltanicaa** [reply]
Oh, not third*
Second topic
